test/convert_to_iso_string.test.ts

```diff
@@ -55,7 +56,7 @@ describe("convertToIsoString", () => {
     new Date("hello world"),
     NaN,
   ])("should return null for invalid ISO string: %s", (invalidDate) => {
-    const converted = convertToIsoString(invalidDate);
+    const converted = convertToIsoString(invalidDate as any);
```

Comment:
Could add a comment on why the typecasting is needed (Because it's supposed to be incorrect types included)